node/forcedroid.js

```diff
@@ -280,15 +280,14 @@ function createNativeOrReactNativeApp(config) {
     if (config.usesmartstore) {
         console.log('Adding SmartStore/SmartSync support.');
         console.log('Extending SmartSyncSDKManager instead of SalesforceSDKManager.');
-        shelljs.sed('-i', /SalesforceSDKManager/g, 'SmartSyncSDKManager', appClassPath);
-        shelljs.sed('-i',
-            /com\.salesforce\.androidsdk\.app\.SmartSyncSDKManager/g,
+        miscUtils.replaceTextInFile(appClassPath, 'SalesforceSDKManager', 'SmartSyncSDKManager');
+        miscUtils.replaceTextInFile(appClassPath,
```

Comment:
It will replace all occurrences of the text in the file. The function splits the file into lines and applies the replacement text to all occurrences of the text to be replaced in that file.